celery/app/backends.py

```diff
@@ -34,6 +34,7 @@
     'azureblockblob': 'celery.backends.azureblockblob:AzureBlockBlobBackend',
     'arangodb': 'celery.backends.arangodb:ArangoDbBackend',
     's3': 'celery.backends.s3:S3Backend',
+    'gs': 'celery.backends.gcs:GCSBackend',
```

Comment:
@haimjether @thedrow something I was wondering about during upgrade-review for `celery` - is this meant to be registered as `gs` or is that a typo of `gcs`?

Reply:
@jayaddison , it was meant to be registered as `gs` , to mimic the blobs path prefix on GCS. E.g 
`gs://my-bucket/my-folder` 

IMHO this is similar to AWS `s3` one.

Reply:
:+1: great, glad to know it's as-intended.  Thanks @haimjether!